app/jobs/v3/delete_route_binding_job.rb

```diff
@@ -38,7 +38,9 @@ def resource_type
 
       def perform
         binding = route_binding
-        gone! unless binding
+        if binding.nil?
+          return finish
+        end
```

Comment:
While writing the tests I noticed we are failing when binding is not there. Given this is a delete should we just succeed the job?

Reply:
Yes. Technically it should go into the job right (as it should exit from the controller), but makes no harm in adding this here